architecture/README.md

```diff
@@ -33,127 +30,135 @@ types of roles. The TiDB community has the following organizations:
 
 ## TiDB Developer Group
 
-### Contributor
+As the cornerstone of community development, the TiDB Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB.
 
-A Contributor refers to the person who contributes to the following projects:
+### Contributor
 
-* TiDB: https://github.com/pingcap/tidb
-* Parser: https://github.com/pingcap/parser
-* TiKV: https://github.com/tikv/tikv
-* TiSpark: https://github.com/pingcap/tispark
-* PD: https://github.com/pingcap/pd
-* Docs: https://github.com/pingcap/docs
-* Docs-cn: https://github.com/pingcap/docs-cn
+New contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
 
-#### How to become a Contributor?
+#### How to become a Contributor
 
-If a PR (Pull Request) submitted to any one of the above projects by you is
-approved and merged, then you become a Contributor.
+- merged at least 1 PR for any project in the [PingCAP Organization](https://github.com/pingcap)
 
 You are also encouraged to participate in the projects in the following ways:
 
-* Actively answer technical questions asked by community users in github issues.
-* Help test the projects.
-* Help review the pull requests (PRs) submitted by others.
-* Help improve technical documents.
-* Submit valuable issues.
-* Report or fix known and unknown bugs.
-* Participate in the existing discussion about features in the roadmap, and
-  have interest in implementing a certain feature independently.
-* Write articles about source code analysis and usage cases for the
-  projects.
+- Actively answer technical questions asked by community users in GitHub issues.
+- Help test the projects
+- Help review the pull requests (PRs) submitted by others
+- Help improve technical documents
+- Submit valuable issues
+- Report or fix known and unknown bugs
+- Write articles about source code analysis and usage cases for the - projects.
+
+#### Responsibilities and privileges
+
+- Become a member in the SIG
+- Get the profile listed on the membership page of the TiDB community
 
 #### Contributor List
 
-[TiDB Contributors](./contributor-list.md)
+See [TiDB Contributors](./contributor-list.md)
 
 ### Active Contributor
 
-TBD
+Active Contributors are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in SIGs through GitHub teams, and pre-submit tests are automatically run for their PRs. See `member-list.md` under the specific SIG for a list of Active Contributors.
```

Comment:
> They can have issues and PRs assigned to them

I think everyone can be assigned to an issue, no matter whether they're contributors.

Reply:
Basically I don't think Contributors are not established or committed enough to be assigned with tasks. They are contributing out of their willingness or interest. You can suggest or recommend, but not assign  a task to them (assigning means obligations or responsibilities, which they do not have yet )